Source/Core/AudioCommon/CubebUtils.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <cstdarg>
+#include <cstddef>
+#include <cstring>
+
+#include "AudioCommon/CubebUtils.h"
+#include "Common/CommonPaths.h"
+#include "Common/Logging/Log.h"
+#include "Common/Logging/LogManager.h"
+#include "Common/StringUtil.h"
+
+#include <cubeb/cubeb.h>
+
+static ptrdiff_t s_path_cutoff_point = 0;
+
+static void LogCallback(const char* format, ...)
```

Comment:
@goldenercobra If this were isolated code, that would probably be ideal, yeah. But this needs to be passed to `cubeb_set_log_callback` as a pointer of the type

```c++
typedef void (* cubeb_log_callback)(char const * fmt, ...);
```